src/EjectionManager.sol

```diff
@@ -83,11 +83,18 @@ contract EjectionManager is IEjectionManager, OwnableUpgradeable{
                     quorumEjectionParams[quorumNumber].rateLimitWindow > 0 &&
                     stakeForEjection + operatorStake > amountEjectable
                 ){
-                    stakeEjectedForQuorum[quorumNumber].push(StakeEjection({
-                        timestamp: block.timestamp,
-                        stakeEjected: stakeForEjection
-                    }));
                     ratelimitHit = true;
+
```

Comment:
In the above, if statement, it feels like we could simplify the logic flow by inverting the condition and pulling out the `rateLimitWindow` check to a guard clause

`

            uint32 numEjectedOperators;

            bool ratelimitHit;
            for(uint8 j = 0; j < _operatorIds[i].length; ++j) {
                uint256 operatorStake = stakeRegistry.getCurrentStake(_operatorIds[i][j], quorumNumber);
                if(quorumEjectionParams[quorumNumber].rateLimitWindow == 0) {
                    break;
                }
                if (isEjector[msg.sender] && !ratelimitHit) {
                    stakeForEjection += operatorStake;
                    ++numEjectedOperators;
                    registryCoordinator.ejectOperator(
                        registryCoordinator.getOperatorFromId(_operatorIds[i][j]),
                        abi.encodePacked(quorumNumber)
                    );

                    emit OperatorEjected(_operatorIds[i][j], quorumNumber);

                    if (stakeForEjection > amountEjectable) {
                        stakeEjectedForQuorum[quorumNumber].push(StakeEjection({
                            timestamp: block.timestamp,
                            stakeEjected: stakeForEjection
                        }));
                        break;
                    }
                }
            }

            emit QuorumEjection(numEjectedOperators, ratelimitHit);
        }
`